distributed/dashboard/components/scheduler.py

```diff
@@ -1856,7 +1856,10 @@ def update(self):
                 )
                 continue
             try:
-                data[name].insert(0, sum(data[name]))
+                total_data = sum(data[name])
+                if name == "memory_percent":
+                    total_data /= len(data[name])
```

Comment:
I don't actually think this is right if different workers have different memory limits. That is probably not super common, but it's probably more correct to do: 

```suggestion
                    n = len(data[name])
                    total_data = sum(data["memory"][-n:]) / sum(data["memory_limit"][-n:])
```